installation/resources/kyma/installer-overrides-compass-gateway.yaml

```diff
@@ -7,6 +7,7 @@ metadata:
     installer: overrides
     kyma-project.io/installation: ""
 data:
-  global.istio.gateway.name: "compass-istio-gateway"
-  global.istio.gateway.namespace: "compass-system"
+  global.istio.gateway.name: "kyma-gateway"
```

Comment:
This are the same values as the defaults from kyma we can remove the whole file.

Reply:
The file is removed